.github/workflows/platformDeploy.yml

```diff
@@ -352,6 +352,16 @@ jobs:
         env:
           CF_API_KEY: ${{ secrets.CLOUDFLARE_TOKEN }}
 
+  # Build a version of iOS and Android HybridApp if we are deploying to staging
+  hybridApp:
+    runs-on: ubuntu-latest
+    needs: validateActor
+    if: ${{ fromJSON(needs.validateActor.outputs.IS_DEPLOYER) }}
+    steps:
+      - name: 'Deploy HybridApp'
+        if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
```

Comment:
Why did you move this if to a step level instead of the job level?

Reply:
I had to move the entire job to a step for the tests to pass: https://github.com/Expensify/App/pull/42166#issuecomment-2142674087

I just copied the existing step level `if` from all the other steps, but happy to adjust if there is something better

Reply:
kind of lame that the limitations in the tests are driving the workflow design, but in this case I guess it doesn't matter too much. This looks like it should work.